src/main/java/io/confluent/connect/jdbc/JdbcUtils.java

```diff
@@ -154,5 +155,17 @@ public static String getAutoincrementColumn(Connection conn, String table) throw
   public static String formatUTC(Date date) {
     return DATE_FORMATTER.get().format(date);
   }
+
+  /**
+   * Get the string used for quoting identifiers in this database's SQL dialect.
+   * @param connection the database connection
+   * @return the quote string
+   * @throws SQLException
+   */
+  public static String getIdentifierQuoteString(Connection connection) throws SQLException {
+    String quoteString = connection.getMetaData().getIdentifierQuoteString();
```

Comment:
Neat! I didn't realize you can get this from the DB. I always made users configure this...